docs/rst/types/json_tags.rst

```diff
@@ -0,0 +1,23 @@
+.. include:: ../exports/alias.include
+
+.. _types_tags:
+
+JSON Tags
+=========
+
+The |StatisticsBackend-api| use JSON format to retrieve information in many methods as |get_info-api|,
+|get_graph-api| or |dump_database-api|.
+
+.. todo::
+
+    In order to access every item in every JSON generated, the following tags are provided:
+    Create table
+
+Dump Tags Example
+-----------------
```

Comment:
```suggestion
Database Dump Tags Example
--------------------------

Reply:
In my opinion the Database should not be known by a user